### PR TITLE
ACC-18 ACC-18 Set MaxWorkplaceSpace and WorkspaceCapacity endpoint Ge…

### PR DESCRIPTION
**Pull request by szukacs**
…neral info endpoint Capacity service

### CODE REVIEW

**Review by mrbelpit — APPROVED** [review]

File: src/main/java/accenture/demo/capacity/CapacityHandler.java [hunk 1]
```diff
@@ -28,24 +27,32 @@ public static CapacityHandler getInstance() {
     return instance;
   }
 
-  public void addUser(AppUser user) {
+  public String registerAppUser(AppUser user) {
     if (!allowedUsers.offer(user)) {
       userQueue.add(user);
+      return "Currently all place is booked. Your current place in the queue " + userQueue.size()
```
Comment (spiglebach) [inline src/main/java/accenture/demo/capacity/CapacityHandler.java]: I think "Currently all place is booked." is unnecessary.
Suggestion:
modify to: Your current place in the queue: *number*

File: src/main/java/accenture/demo/capacity/CapacityHandler.java [hunk 2]
```diff
@@ -28,24 +27,32 @@ public static CapacityHandler getInstance() {
...
+          + "!";
+    }
+    return "You can enter today!";
```
Comment (spiglebach) [inline src/main/java/accenture/demo/capacity/CapacityHandler.java]: Suggestion:
modify to: You can enter the office!

File: src/main/java/accenture/demo/capacity/CapacityHandler.java [hunk 3]
```diff
@@ -1,22 +1,21 @@
...
 
   private static CapacityHandler instance;
   private Integer maxWorkplaceSpace = 250;
   private Double workspaceCapacity = 0.1;
-  private Queue<AppUser> allowedUsers = new LinkedBlockingQueue<>(
+  private LinkedBlockingQueue<AppUser> allowedUsers = new LinkedBlockingQueue<>(
       (int) (maxWorkplaceSpace * workspaceCapacity));
   private Queue<AppUser> userQueue = new LinkedList<>();
-
+  private ArrayList<AppUser> currentlyWorkerIn = new ArrayList<>();
```
Comment (spiglebach) [inline src/main/java/accenture/demo/capacity/CapacityHandler.java]: Suggestion:
modify to: usersCurrentlyInOffice or usersInOffice

File: src/main/java/accenture/demo/capacity/CapacityInfoDTO.java [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class CapacityInfoDTO {
+
+  private Integer maxWorkplaceSpace;
+  private Integer workspaceCapacity;
+  private Integer maxWorkerAllowedToEnter;
+  private Integer workerInTheBuildingCurrently;
```
Comment (spiglebach) [inline src/main/java/accenture/demo/capacity/CapacityInfoDTO.java]: Suggestion:
modify to: workersCurrentlyInOffice

**Review by spiglebach — CHANGES_REQUESTED** [review]

**Review by spiglebach — APPROVED** [review]

**Review by spiglebach — APPROVED** [review]

**Pull request merged**